api/src/permissions/modules/process-ast/utils/flatten-filter.test.ts

```diff
@@ -8,7 +8,7 @@ test('Returns early when no filter is passed', () => {
 
 	flattenFilter(paths, undefined);
 
-	expect(paths).toBe(paths);
+	expect(paths).toEqual([]);
```

Comment:
This check did not make sense before, since `flattenFilter` never changes the reference to paths (how could it) so it will always be the same object, no matter what `flattenFilter` does to it, so `[]` is more correct.